iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Boundary.java

```diff
@@ -18,4 +18,12 @@
     double getP();
 
     double getQ();
+
+    default Branch.Side getSide() {
```

Comment:
You need this function to be able to build a terminal ref or something similar?

Reply:
Boundary can refers to a TieLine's boundary, in which case the user should know which "side" they are looking at